server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -134,7 +134,7 @@ public Repo<Manager> call(long tid, Manager manager) throws Exception {
             if (key.getColumnFamily().equals(DataFileColumnFamily.NAME)) {
               StoredTabletFile exportedRef;
               var dataFileCQ = key.getColumnQualifier().toString();
-              if (tableInfo.exportedVersion == null || (tableInfo.exportedVersion < VERSION
```

Comment:
When `tableInfo.exportedVersion < VERSION_2` is true under what circumstance would we expect `StoredTabletFile.fileNeedsConversion(dataFileCQ)` to return false?

Reply:
Removed extra check in 8191010f6